script.js

```diff
@@ -0,0 +1,200 @@
+let webBrands = [];
+let brands=[];
+let resultbrands=[];
+let productNode;
+const proxyurl = "https://cors-anywhere.herokuapp.com/";
+const url = `https://wellspent-ethical.herokuapp.com/api/brands`;
+
+
+var linksList = document.querySelectorAll("[id^=product-]");
+linksList.forEach(function(product) {
+  const node = product.childNodes[0];
+  const label = node.getAttribute("aria-label");
+  let brand = label.toLowerCase();
+  webBrands.push(brand);
+});
+
+
+ // site that doesn’t send Access-Control-*
+fetch(proxyurl + url) // https://cors-anywhere.herokuapp.com/https://example.com
+  .then(resultbrandsonse => resultbrandsonse.text())
+  .then(contents => {
+    resultbrands = JSON.parse(contents);
+    brands = resultbrands;
+
+
+    const finalResult = brands.result.map((data)=>{
```

Comment:
- Using destructuring objects will be more clear and understandable.  
- Same here for variables names/not clear